### PR TITLE
enhance(): update allowed Facebook API versions

### PR DESCRIPTION
**Pull request by montoyadamien**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,6 @@ dist
 node_modules
 storybook-static
 .vscode
-renovate.json
\ No newline at end of file
+renovate.json
+.idea/
+yarn.lock
```
Comment (greatSumini) [inline .gitignore:8]: Can you exclude yarn.lock?
We have to track dependency changes!

**Review by greatSumini — COMMENTED** [review]
Thanks for first contribute ☺️☺️🙇🙇

File: .gitignore (hunk 1 above)
Comment (montoyadamien) [reply .gitignore]: Done !

**Review by greatSumini — APPROVED** [review]
LGTM 🥇

**Pull request merged**